tools/xchain-suite/src/pages/SwapPage.tsx

```
@@ -586,7 +581,7 @@ export default function SwapPage() {
       )}
 
       {/* Code Examples */}
-      {fromAsset && toAsset && amount && (
+      {fromAsset && toAsset && amount && selectedQuote?.protocol !== 'Chainflip' && (
```

ESLint

[error] Insert `··`

(prettier/prettier)